Editor/Editors/CinemachineVirtualCameraBaseEditor.cs

```diff
@@ -13,6 +14,12 @@ namespace Cinemachine.Editor
     public class CinemachineVirtualCameraBaseEditor<T>
         : BaseEditor<T> where T : CinemachineVirtualCameraBase
     {
+        public static class Styles
+        {
+            public static GUIContent addExtensionLabel = new GUIContent("Add Extension");
+            public static GUIContent virtualCameraChildrenInfoMsg = new GUIContent("The Virtual Camera Children field displays the child objects of a given parent object, thus this field is not available when multiple objects are selected.");
```

Comment:
This text will need to be fixed not to display "Virtual Camera" for cases when it's on a different type of camera.